tools/viewsvg/mainwindow.ui

```diff
@@ -151,10 +145,10 @@
        </sizepolicy>
       </property>
       <property name="frameShape">
-       <enum>QFrame::StyledPanel</enum>
+       <enum>QFrame::NoFrame</enum>
```

Comment:
This and the next change remove all border from the area, allowing an elegant borderless window